packages/expo-router/src/fork/__tests__/getStateFromPath.test.web.ts

```diff
@@ -100,19 +100,9 @@ describe(getUrlWithReactNavigationConcessions, () => {
       );
     });
   });
-
-  [
-    ['', ''],
-    ['https://acme.com/hello/world/?foo=bar#123', 'https://acme.com/hello/world/?foo=bar'],
-    ['/foobar#123', '/foobar'],
-  ].forEach(([url, expected]) => {
-    it(`returns the pathname without hash for ${url}`, () => {
-      expect(getUrlWithReactNavigationConcessions(url).inputPathnameWithoutHash).toBe(expected);
-    });
-  });
```

Comment:
These were removed as `inputPathnameWithoutHash` isn't actually used in the code base and we not support hashes